### PR TITLE
Intl staging

### PR DESCRIPTION
**Pull request by Bijnagte**

### CODE REVIEW

File: devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy [hunk 1]
```diff
@@ -107,15 +118,17 @@ private handleBatteryEvent(volts) {
             volts = minVolts
         else if (volts > maxVolts)
             volts = maxVolts
-        def pct = batteryMap[volts]
-        if (pct != null) {
+        def value = batteryMap[volts]
+        if (value != null) {
             def linkText = getLinkText(device)
+            descriptionText = '{{ linkText }} battery was {{ value }}'
```
Comment (Bijnagte) [inline devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy:124]: this had the % removed for some reason

File: devicetypes/smartthings/smartpower-outlet.src/smartpower-outlet.groovy [hunk 2]
```diff
@@ -91,22 +99,24 @@ def parse(String description) {
...
-		log.info finalResult
+		log.info "final result = $finalResult"
 		if (finalResult.type == "update") {
 			log.info "$device updates: ${finalResult.value}"
 		}
 		else if (finalResult.type == "power") {
-			def powerValue = (finalResult.value as Integer)/10
-			sendEvent(name: "power", value: powerValue)
+			def value = (finalResult.value as Integer)/10
+			createEvent(name: "power", value: value, descriptionText: '{{ device.displayName }} power is {{ value }} Watts', translatable: true )
```
Comment (Bijnagte) [inline devicetypes/smartthings/smartpower-outlet.src/smartpower-outlet.groovy:108]: not sure why this is createEvent now?

File: devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy [hunk 3]
```diff
@@ -245,7 +256,8 @@ private Map getBatteryResult(rawValue) {
 				def pct = batteryMap[volts]
 				if (pct != null) {
 					result.value = pct
-					result.descriptionText = "${linkText} battery was ${result.value}%"
+                    def value = pct
+					result.descriptionText = "{{ device.displayName }} battery was {{ value }}"
```
Comment (Bijnagte) [inline devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy:260]: no % here too

File: devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy [hunk 4]
```diff
@@ -62,13 +71,13 @@ metadata {
...
+			input title: "Temperature Offset", description: "This feature allows you to correct any temperature variations by selecting an offset. Ex: If your sensor consistently reports a temp that's 5 degrees too warm, you'd enter '-5'. If 3 degrees too cold, enter '+3'.", displayDuringSetup: false, type: "paragraph", element: "paragraph"
 			input "tempOffset", "number", title: "Degrees", description: "Adjust temperature by this many degrees", range: "*..*", displayDuringSetup: false
 		}
 		section {
-			input("garageSensor", "enum", title: "Do you want to use this sensor on a garage door?", options: ["Yes", "No"], defaultValue: "No", required: false, displayDuringSetup: false)
+ 			input("garageSensor", "enum", title: "Do you want to use this sensor on a garage door?", translatable: true, description: "Tap to set", options: ["Yes","No"], defaultValue: "No", required: false, displayDuringSetup: false)
```
Comment (Bijnagte) [inline devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy:78]: no idea why this is translatable now? input does not do anything with a translatable option

File: devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy (hunk 1 above)
Comment (twack) [inline devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy:124]: % char was breaking translation

File: devicetypes/smartthings/smartpower-outlet.src/smartpower-outlet.groovy (hunk 2 above)
Comment (twack) [inline devicetypes/smartthings/smartpower-outlet.src/smartpower-outlet.groovy:108]: could be either.

File: devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy (hunk 4 above)
Comment (twack) [inline devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy:78]: true, was trying that before input was translating correctly.

File: devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy (hunk 3 above)
Comment (twack) [inline devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy:260]: again, "%" char broke translation

File: devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy (hunk 1 above)
Comment (Bijnagte) [inline devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy:124]: ok, that issue has been resolved so we should be able to add it back

**Pull request closed without merging**